sdks/python/apache_beam/runners/portability/stager_test.py

```diff
@@ -222,6 +223,8 @@ def test_with_requirements_file_and_cache(self):
     self.assertTrue(os.path.isfile(os.path.join(staging_dir, 'abc.txt')))
     self.assertTrue(os.path.isfile(os.path.join(staging_dir, 'def.txt')))
 
+  @unittest.skipIf(sys.version_info[0] == 3, 'This test still needs to be '
+                                             'fixed on Python 3')
```

Comment:
This test does not fail on its own or when only running the runners tests. It only fails when running the full test suite, so it seems like there is a conflict with another test.

Reply:
Filed https://issues.apache.org/jira/browse/BEAM-5502 for this one. Please add a TODO with the Jira in the comment.